### PR TITLE
Fixed initial delay handling

### LINKED ISSUE #865: Health check initial delay is not correctly executed
It seems that if health check initial delay is 10s and interval 60s, checker will wait 70s before checking health... should be 10s.

### PR DESCRIPTION
**Pull request by jnummelin**
fixes #865 

ping @jakolehm

### CODE REVIEW

File: agent/lib/kontena/workers/container_health_check_worker.rb [hunk 1]
```diff
@@ -20,28 +20,35 @@ def initialize(container, queue)
...
-      timeout = labels['io.kontena.health_check.timeout'].to_i
-      port = labels['io.kontena.health_check.port'].to_i
-      protocol = labels['io.kontena.health_check.protocol']
-      ip, _ = @container.overlay_cidr.split('/')
+      initial_delay = @container.labels['io.kontena.health_check.initial_delay'].to_i
+      interval = @container.labels['io.kontena.health_check.interval'].to_i
 
       sleep initial_delay # to allow container to startup properly
 
+      check_status()
```
Comment (nevalla) [inline agent/lib/kontena/workers/container_health_check_worker.rb]: 🔎 redundant brackets here

File: agent/lib/kontena/workers/container_health_check_worker.rb [hunk 2]
```diff
@@ -20,28 +20,35 @@ def initialize(container, queue)
...
+      
+      uri = @container.labels['io.kontena.health_check.uri']
+      timeout = @container.labels['io.kontena.health_check.timeout'].to_i
+      port = @container.labels['io.kontena.health_check.port'].to_i
+      protocol = @container.labels['io.kontena.health_check.protocol']
+      ip, _ = @container.overlay_cidr.split('/')
+
+      if protocol == 'http'
+        msg = check_http_status(ip, port, uri, timeout)
+      elsif
```
Comment (nevalla) [inline agent/lib/kontena/workers/container_health_check_worker.rb]: should be `else`?

Comment (jnummelin) [inline agent/lib/kontena/workers/container_health_check_worker.rb]: yes, good catch :)

**Comment by jakolehm** [comment]
LGTM!

**Pull request merged**